doc/drawingElement.md

```diff
@@ -0,0 +1,1186 @@
+
+# Elements to setup canvas (drawing space)
+
+## `pl-drawing` element
+
+A `pl-drawing` element displays a canvas, which can be used to display initial drawing objects or expect drawing objects for grading.
+
+The system of coordinates of the canvas is located at the top/left corner.
+
+#### Sample Element
+
+```html
+<pl-drawing answers-name="fbd">
+</pl-drawing>
+```
+<img src="elements/pl-drawing.png" width=70%>
+
+#### Customizations
+
+Attribute | Type | Default | Description
+--- | --- | --- | ---
+`preview` | boolean | false | `preview = true` is used when canvas is used for display only. `preview = false` expects objects to be placed in the canvas for grading.
```

Comment:
Should we agree on a name before we make a change? What about `gradable` and set it to `false` by default?

Reply:
`gradable` (default `false`) sounds excellent to me!